package.json

```
@@ -4,13 +4,14 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "heroku-prebuild": "cd client && npm ci",
```

```suggestion
    "heroku-prebuild": "cd client && npm i",
```

just like in this [pull](https://github.com/lmachens/copypasta/pull/52/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2)

---

SRY! That's only if you use cached node_modules on heroku...